components/automate-cli/cmd/chef-automate/license.go

```diff
@@ -402,19 +406,27 @@ func init() {
 	licenseCmd.AddCommand(complianceResourceUsageCmd)
 	uniqueNodeCounterCmd.Flags().StringVarP(&CommandFlags.StartTime, "start_time", "s", "", startTimeFormat)
 	uniqueNodeCounterCmd.Flags().StringVarP(&CommandFlags.EndTime, "end_time", "e", "", endTimeFormat)
-	uniqueNodeCounterCmd.Flags().StringVarP(&CommandFlags.ESHostname, "es_hostname", "n", "localhost", hostnameES)
-	uniqueNodeCounterCmd.Flags().StringVarP(&CommandFlags.ESPort, "es_port", "p", "10144", portES)
+	uniqueNodeCounterCmd.Flags().StringVarP(&CommandFlags.ESHostname, "os_hostname", "n", "localhost", hostnameES)
```

Comment:
can we use the short letter `h` instead of `n`? If we want to maintain similarity with other existing commands we can continue with `n` only.

Reply:
`h` is reserved for help. Also this is already available in production. Can't change now since it will break the usability.